src/Simulator/IncrementalRedstoneSimulator/ObserverHandler.h

```diff
@@ -0,0 +1,106 @@
+
+#pragma once
+
+#include "RedstoneHandler.h"
+#include "../../Blocks/BlockObserver.h"
+
+
+
+
+
+class cObserverHandler : public cRedstoneHandler
+{
+public:
+
+	inline static bool IsOn(NIBBLETYPE a_Meta)
+	{
+		return (a_Meta & 0x8) == 0x8;
```

Comment:
You yourself voted to use byte-sized hex constants, so use them here as well :)

Reply:
Metadata is explicitly a nybble, so I'm not sure if that makes sense here (that said, I haven't seen the relevant vote).  Well, in versions where metadata still exists, at least.